### DISCUSSION TITLE
RGB-21 Feedback: Engraving & Reserves

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/LNPBPs#80] [Ideas]
Received from @ctramount

From my understanding of the proposed engraving and reserve features, both open the potential to interesting use cases.

Engraving sounds like a native way to enable async.art 's model of minting "master NFTs" which are composed of several "layer NFTs". Layer NFTs can be modified by owners which in turn alters the master NFT. The ability to collaborate on a single work seems to have found traction among the NFT art community - async.art recently set a record with their $120K sale for a single work (https://www.kogocrypto.com/micah-johnsons-nft-artwork-sells-for-120000-on-async-art/).

The reserve feature could be immediately useful to Scarce.City. Our first planned NFT use case is to associate tokens with physical art as a tokenized certificate of authenticity. Reserves could allow us to automatically and trustlessly transfer the token to the buyer of the physical work when they make a payment to a designated Bitcoin address.

I think these use cases are just scratching the surface of what these features enable.

**Comment by dr-orlovsky** [comment]
Resolved as a part of RGB21 interface: https://github.com/LNP-BP/LNPBPs/pull/137
